### PR TITLE
adapt documentation on how to integrate with alpine.js V3

### PR DESCRIPTION
**Pull request by grrrisu**
Alpine.js introduces some breaking changes with their latest version 3. 
This PR adapts the documentation on how to integrate live view with alpine.js.

### CODE REVIEW

**Comment by Dtatoo** [comment]
Can we add `if (from.nodeType !== 1) return` check just like what livewire is doing here?
https://github.com/livewire/livewire/blob/a4ffb135693e7982e5b982ca203f5dc7a7ae1126/js/component/SupportAlpine.js#L287-L296

**Comment by dvic** [comment]
> Can we add `if (from.nodeType !== 1) return` check just like what livewire is doing here?
> https://github.com/livewire/livewire/blob/a4ffb135693e7982e5b982ca203f5dc7a7ae1126/js/component/SupportAlpine.js#L287-L296

Probably, but I'm not 100% sure if the same logic applies to LiveView, here's what the '1' means:
https://github.com/livewire/livewire/blob/6621b7fc4feecd5f4e68c04fd612f22043dfcf50/js/dom/morphdom/morphdom.js#L16-L19

**Comment by mcrumm** [comment]
The nodeType check appears redundant to the property check, but perhaps I am missing something.

// @calebporzio

**Comment by mcrumm** [comment]
re: the nodeType check, [the general consensus is we don't need it](https://github.com/livewire/livewire/discussions/3279) :)

I think this is good as-is, thanks!

File: guides/client/js-interop.md [hunk 1]
```diff
@@ -188,7 +188,9 @@ and the return value is ignored. For example, the following option could be used
       ...,
       dom: {
         onBeforeElUpdated(from, to){
-          if(from.__x){ window.Alpine.clone(from.__x, to) }
+          if(from._x_dataStack){
+            window.Alpine.clone(from, to)
+          }
```
Comment (mcrumm) [inline guides/client/js-interop.md:193]:
```suggestion
          if(from._x_dataStack){ window.Alpine.clone(from, to) }
```

**Pull request merged**